### PR TITLE
Add day 4 project files and codes in the HTML CSS file

### PR DESCRIPTION
**Pull request by yash244466666**
Added following new files from the design template

- [skill-dot.svg](https://github.com/yash244466666/Microverse---Portfolio-setup-and-mobile-first/blob/day4htmlcss/res/skill-dot.svg)
- [language-dot.svg](https://github.com/yash244466666/Microverse---Portfolio-setup-and-mobile-first/blob/day4htmlcss/res/language-dot.svg)
- [farmework-dot.svg](https://github.com/yash244466666/Microverse---Portfolio-setup-and-mobile-first/blob/day4htmlcss/res/farmework-dot.svg)
- [img_Placeholder.png](https://github.com/yash244466666/Microverse---Portfolio-setup-and-mobile-first/blob/day4htmlcss/res/img_Placeholder.png)
- [img_Placeholder _2.png](https://github.com/yash244466666/Microverse---Portfolio-setup-and-mobile-first/blob/day4htmlcss/res/img_Placeholder%20_2.png)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,6 +27,139 @@
...
+                <div class="project-1">
+                    <img class="project-1-img" src="res/img_Placeholder.png">
+                    <h3>Multi-Post Stories</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <div class="siklls">
+                        <a>css</a>
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
```
Comment (tresorsawasawa) [inline index.html]: - [x] Kindly make sure the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") use a `ul` tag as a container and each of them a `li` tag for best semantics tags and as required for the project
**Make sure you apply this approach on all cards**

File: index.html [hunk 2]
```diff
@@ -27,6 +27,139 @@
...
+                    <div class="project-2-text">
+                        <h3>Profesional Art </br> Printing Data</h3>
+                        <p>A daily selection of privately </br> personalized reads; no accounts or </br> sign-ups required. has been the </br> industry's standard</p>
+                    </div>
+
+                    <div class="siklls skills2">
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
```
Comment (tresorsawasawa) [inline index.html]: - [x] Kindly make sure all the cards' design sticks to the form of the Figma template e.g.: spacing between cards, See Project button width, fons size on the project detail text,... You can check all the details from Figma to match your design.

| Figma design | Your Design |
| -------------------- | ------------------- | 
| Screenshot from 2022-09-24 12-30-31 | Screenshot from 2022-09-24 12-31-18 |

File: index.html [hunk 3]
```diff
@@ -27,6 +27,139 @@
...
+                    <button class="project-2-btn">See Project</button>
+                </div>
```
Comment (tresorsawasawa) [inline index.html]: - [x] Kindly make sure  all the buttons (e.g., "See project" &&  "Gey My Resume")) follow the button styling: Active, Hover, and Pressed (see design) like on Figma. You check all the colors and fonts used on this image below from the Figma template.
Screenshot from 2022-09-24 12-41-27

File: index.html [hunk 4]
```diff
@@ -27,6 +27,139 @@
...
+            </div>
+        </section>
+        <section class="about-myself">
+            <div class="aboutme">
+                <h1 class="about-h1">About me</h1>
+                <p class="about-p">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button class="about-btn">Get My Resume</button>
+                <hr>
+            </div>
```
Comment (tresorsawasawa) [inline index.html:133]: - [x] Kindly ensure the `Get My Resume button` follows the design like the Figma template. Below is the image of your page on screen width: `375px - 445px`.
**Also, remember to add the spacing at the bottom of the button.**

| Figma design | Your Design |
| --------------- | ------------------- |
| Screenshot from 2022-09-24 12-48-24 | Screenshot from 2022-09-24 12-47-04 |

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @yash244466666 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yash244466666** [comment]
@tresorsawasawa  I've made all the changes you instructed.  I tried to keep sticking to the Figma design.

**Review by cynthiainga — APPROVED** [review]
Hi @yash244466666, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [x] No linters errors ✔️ 
- [x] Good commit history ✔️ 
- [x] Descriptive PR ✔️ 
- [x] Follow correct Github flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**